base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eu
+
+target_paths=$@
+
+
+get_used_epics_modules() {
+    targets=$@
+
+    executables=$(find $targets -type f -executable -print)
```

Comment:
`-print` is the default, and I had to search for it to see if it changed anything and confirm it's the default. I'd rather omit it.